go/pkg/rexec/rexec.go

```diff
@@ -261,7 +261,7 @@ func (ec *Context) ExecuteRemotely() {
 	log.V(1).Infof("%s> Checking inputs to upload...", cmdID)
 	// TODO(olaola): compute input cache hit stats.
 	ec.Metadata.EventTimes[command.EventUploadInputs] = &command.TimeInterval{From: time.Now()}
-	err := ec.client.GrpcClient.UploadIfMissing(ec.ctx, ec.inputBlobs...)
+	_, err := ec.client.GrpcClient.UploadIfMissing(ec.ctx, ec.inputBlobs...)
```

Comment:
It would be really nice if you add this stat to the command Metadata here as well :-) We can also file a bug and do that later.

Reply:
Done. Note that I was a bit lazy and just ignored this field in the test comparison... Otherwise it pulled in the digests of `Action` and the command, which is a bit cumbersome to set up...